### PR TITLE
new site blog post

### PR DESCRIPTION
**Pull request by zeke**
https://electron-website-pr-835.herokuapp.com/blog/new-website

### CODE REVIEW

File: data/blog/new-website.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+Electron has a new website at [electronjs.org]!  We've replaced 
+our static Jekyll site with a Node.js webserver, giving us flexibility to
+internationalize the site and paving the way for more exciting new features.
+
+## 🌍 Translations
+
+We've begun the process of internationalizing the website with the 
+goal of making Electron app development accessible to a global audience of 
+developers. We're using a localization platform called [Crowdin] that integrates 
+with  GitHub, opening and updating pull requests automatically as content is translated into different languages.
```
Comment (sethlu) [inline data/blog/new-website.md]: Extra whitespace

File: data/blog/new-website.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
+---
+title: "Electron's New Internationalized Website"
+author: zeke
+date: '2017-11-13'
+---
+
+Electron has a new website at [electronjs.org]!  We've replaced
```
Comment (sethlu) [inline data/blog/new-website.md]: Extra whitespace between words

File: data/blog/new-website.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+organically and joined in the effort to internationalize the website and 
+translate Electron's docs into over 20 languages. We are seeing [daily 
+contributions](https://github.com/electron/electron-i18n/pulls?utf8=%E2%9C%93&q=is%3Apr%20author%3Aglotbot%20) from people all over the world, with translations for 
+languages like French, Vietnamese, Indonesian, and Chinese leading the way.
+
+To choose your language and view translation progress, visit [electronjs.org/languages](https://electronjs.org/languages)
+
+<figure>
+  <a href="https://electronjs.org/languages">
+    <img class="screenshot"  src="https://user-images.githubusercontent.com/2289/32754734-e8e43c04-c886-11e7-9f34-f2da2bb4357b.png" alt="Current target languages on Crowdin">
```
Comment (sethlu) [inline data/blog/new-website.md]: Extra whitespace

File: data/blog/new-website.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+As of today, any Electron app can easily have its own page on the Electron 
+site. For a few examples, check out 
+[Etcher](https://electronjs.org/apps/etcher),
+[1Clipboard](https://electronjs.org/apps/1clipboard), or 
+[GraphQL Playground](https://electronjs.org/apps/graphql-playground), pictured
+here on the Japanese version of the site:
+
+<figure>
+  <a href="https://electronjs.org/apps/graphql-playground">
+    <img class="screenshot"  src="https://user-images.githubusercontent.com/2289/32871096-f5043292-ca33-11e7-8d03-a6a157aa183d.png" alt="GraphQL Playground">
```
Comment (sethlu) [inline data/blog/new-website.md]: Extra whitespace

File: data/blog/new-website.md [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+that makes it easy to install desktop apps using a single command in your
+terminal, like `brew cask install atom`.
+
+We've begun collecting Homebrew cask names for popular Electron apps and are now
+displaying the installation command (for macOS visitors) on every app page 
+that has a cask:
+
+<figure>
+  <a href="https://electronjs.org/apps/dat">
+   <img  class="screenshot" src="https://user-images.githubusercontent.com/2289/32871246-c5ef6f2a-ca34-11e7-8eb4-3a5b93b91007.png">
```
Comment (sethlu) [inline data/blog/new-website.md]: Extra whitespace

Not sure about the formatting for indented lines... Probably inserting an extra space before `<img...`? Same with next line.

File: data/blog/new-website.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+
+## 🙏 Thanks to Contributors
+
+We'd like to give special thanks to all the folks around the world who have 
+contributed their own time and energy to help improve Electron. The passion of 
+the open-source community has helped immeasurably in making Electron a success.
+Thank you!
+
+<figure>
+   <img src="https://user-images.githubusercontent.com/2289/32871386-92eaa4ea-ca35-11e7-9511-a746c7fbf2c4.png">
```
Comment (sethlu) [inline data/blog/new-website.md]: Inconsistency in indentation

File: data/blog/new-website.md [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+built on web technologies. Electron was originally called `atom-shell`. Atom 
+was the first app to use it, but it didn't take long for folks to realize that 
+this magical Chromium + Node runtime could be used for all kinds of different 
+applications. When companies like Microsoft and Slack started to make use of
+`atom-shell`, it became clear that the project needed a new name.
+
+And so "Electron" was born. In early 2016, GitHub assembled a new team to focus
+specifically on Electron development and maintenance, apart from Atom. In the 
+time since, Electron has been adopted by thousands of app developers, and is now
+depenended on by many large companies, many of which have Electron teams of
```
Comment (ckerr) [inline data/blog/new-website.md]: depended

File: data/blog/new-website.md [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+
+Supporting GitHub's Electron projects like Atom and [Desktop] is still a 
+priority for our team, but by moving to a new domain we hope to help clarify 
+the technical distinction between Atom and Electron.
+
+## 🐢🚀 Node.js Everywhere
+
+The previous Electron website was built with [Jekyll], the popular Ruby-based 
+static site generator. Jekyll is a great tool for building static websites, but 
+the website had started to outgrow it. We wanted more dynamic capabilities like proper redirects and dynamic content rendering, so a Node.js server was the obvious choice.
```
Comment (ckerr) [inline data/blog/new-website.md]: Should this be `[Node.js]` with a link below, just as we've done for `[Jekyll]` et al?

File: data/blog/new-website.md [hunk 9]
```diff
@@ -0,0 +1,191 @@
...
+to find, and not every developer has the time or resources to build a proper 
+website to market and distribute their app.
+
+Using just a 
+[PNG icon file and a small amount of app metadata](https://github.com/electron/electron-apps/blob/master/contributing.md),
+we're able to collect a lot of information about a given app.
+Using data collected from GitHub, app pages can now display screenshots, 
+download links, versions, release notes, and READMEs for every app that 
+has a public repository. Using a color palette extracted from each app's icon,
+ we're able to produce [bold and accessible colors](https://github.com/zeke/pick-a-good-color)
```
Comment (ckerr) [inline data/blog/new-website.md]: leading whitespace

File: data/blog/new-website.md [hunk 10]
```diff
@@ -0,0 +1,191 @@
...
+to give each app page some visual distinction.
+
+The [apps index page](https://electronjs.org/apps) now also has categories,
```
Comment (ckerr) [inline data/blog/new-website.md]: No need for a listing comma in a two-item list

File: data/blog/new-website.md (hunk 1 above)
Comment (zeke) [reply data/blog/new-website.md]: Thanks, @sethlu. Fixed the whitespace issues. Any feedback on the actual content?

File: data/blog/new-website.md [hunk 11]
```diff
@@ -0,0 +1,191 @@
...
+</figure>
+
+
+[Crowdin]: https://crowdin.com/project/electron
+[crowdin.com/project/electron]: https://crowdin.com/project/electron
+[electron/electron-apps]: https://github.com/electron/electron-apps
+[electron/electron-i18n]: https://github.com/electron/electron-i18n#readme
+[electronjs.org]: https://electronjs.org
+[Atom]: https://atom.io
+[Destop]: https://desktop.github.com
```
Comment (sethlu) [inline data/blog/new-website.md]: Should be `[Desktop]`, the converter wasn't picking up the link.

File: data/blog/new-website.md [hunk 12]
```diff
@@ -0,0 +1,191 @@
...
+applications. When companies like Microsoft and Slack started to make use of
+`atom-shell`, it became clear that the project needed a new name.
+
+And so "Electron" was born. In early 2016, GitHub assembled a new team to focus
+specifically on Electron development and maintenance, apart from Atom. In the 
+time since, Electron has been adopted by thousands of app developers, and is now
+depenended on by many large companies, many of which have Electron teams of 
+their own.
+
+Supporting GitHub's Electron projects like Atom and [Desktop] is still a
```
Comment (sethlu) [inline data/blog/new-website.md]: Are we referring to **GitHub Desktop**?

Comment (ckerr) [reply data/blog/new-website.md]: "Supporting GItHub's Electron projects..." so yes :smile:

**Comment by zeke** [comment]
Thanks, @ckerr. Made some changes based on your feedback.

File: data/blog/new-website.md (hunk 1 above)
Comment (sethlu) [reply data/blog/new-website.md]: @zeke Content-wise I think it's well written! 👍 Looking forward to seeing it posted soon.

**Pull request merged**